plausible/site.go

```diff
@@ -62,6 +62,22 @@ func (s *Site) CurrentVisitors() (int, error) {
 	return visitors, nil
 }
 
+// Gets details of a site.
```

Comment:
```suggestion
// Get fetches details of a site.
```

Doc comments should always must match the name of the function exactly.